autoscale-core/src/main/java/com/hpe/caf/autoscale/core/ScalerThread.java

```diff
@@ -266,4 +290,37 @@ private int establishMemLimitReached(final double currentMemoryLoad)
         }
         return 0;
     }
+
+    private boolean backoffComplete()
```

Comment:
The function naming is a little confusing here.  It seems strange to have an `isBackoffComplete()` a) without the `is` and b) even though there may never have been any backoff.